arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Anytrust orbit chains keyset generation'
+sidebar_label: 'Keyset Generation for Anytrust Orbits'
+description: 'Keyset generation and setup for Anytrust Orbit chains'
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Anytrust Orbit chains.'
+sidebar_position: 1
+---
+**Anytrust** chains, relies on an external Data Availability Committee (hereafter, "the Committee") to store data and provide it on demand, instead of using the parent chain as Data Availability (DA) layer. 
+
+The Committee comprises N members, among whom it is presumed by AnyTrust that a minimum of two maintain integrity. Consequently, in scenarios where N - 1 members of the Committee pledge to grant access to specific data, it is guaranteed that at least one member among those committing to this promise upholds honesty.
+
+Each member of the Committee is provisioned with a unique pair of BLS public and private keys. It is incumbent upon every member to independently generate a new, secure, and randomly derived set of BLS keys for their exclusive use. To generate a set of BLS keys, you can find more info [here](https://docs.arbitrum.io/node-running/how-tos/running-a-daserver#generate-key) on our docs.
+
+The parent chain is required to be cognizant of the composition of the Committee, encompassing the identity of each member along with their respective public keys, to facilitate verification processes. A 'Keyset' delineates the public keys of all Committee members and stipulates the requisite number of signatures for the legitimacy of a Data Availability Certificate. This configuration of Keysets not only facilitates modifications in the Committee's composition but also empowers members to update their individual key pairs as necessary. For more info, please read <a data-quicklook-from='inside-anytrust'>Inside Anytrust</a> page of our docs.
```

Comment:
(style comment, feel free to ignore)
"cognizant", "encompassing", "facilitate", "delineate", "empower" 
I'd make the the language here simpler and more direct/specific.